### PR TITLE
Add icons to User Actions Summary SFINT-2374

### PR DESCRIPTION
**Pull request by nathanlb**
## What was changed?
Added "duplicate" icon to items in the recently clicked documents list
Added "search" icon to the items in the recent queries list

## How was it tested?
Wrote tests making sure the icon appeared in the HTML of every displayed list item 

## Preview
Screen Shot 2019-09-13 at 1 21 53 PM

### CODE REVIEW

File: src/components/UserActions/UserActions.scss [hunk 1]
```diff
@@ -84,6 +84,12 @@ $search-ui-dropdown-header-color: #1d4f76;
 
         li {
             margin: 0.25em 0;
+
+            .coveo-list-row > * {
+                display: inline;
+                text-overflow: ellipsis;
```
[the remark this answers is not in the document]
Comment (jeremierobert-coveo) [reply src/components/UserActions/UserActions.scss]: Can't see the impact in the given screenshot.

[the remark this answers is not in the document]
Comment (nathanlb) [reply src/components/UserActions/UserActions.scss]: The overflow attributes do not have the desired effect as they're being applied to anchor tags.

[the remark this answers is not in the document]
Comment (jeremierobert-coveo) [reply src/components/UserActions/UserActions.scss]: So remove this css 😄

[the remark this answers is not in the document]
Comment (nathanlb) [reply src/components/UserActions/UserActions.scss]: Done

**Comment by louis-bompart** [comment]
:shipit:

**Pull request merged**